### DISCUSSION TITLE
Add support for `cabbrev`

### DISCUSSION THREAD

**Discussion by powerman** [Wansmer/langmapper.nvim#39] [Ideas]
It's really annoying trying to exit with `:й<CR>` instead of `:q<CR>` just to get `E492: Not an editor command: й`.
This can be solved using something like this:

```vim
cabbrev <expr> й getcmdtype()==':' && getcmdline()=='й' ? 'q' : 'й'
```

But it would be great to have this done automatically for all commands just like it works for all mappings.

**Comment by Wansmer** [comment]
I think the best solution for your case is to change the keyboard layout to English when you switch to command mode. Here is an example of helper functions to set the desired input mode from nvim: https://github.com/Wansmer/nvim-config/blob/f997ad0de33865f9d0bdf2ab8118a625bd8b7fec/lua/utils.lua#L454-L480


There are no plans to add built-in support for abbreviations, because it is impossible to provide all the necessary ones and each user will have a different set of necessary abbreviations.
